waku/v2/node/waku_node.nim

```diff
@@ -148,6 +148,11 @@ proc new*(T: type WakuNode,
           nameResolver: NameResolver = nil,
           sendSignedPeerRecord = false,
           dns4DomainName = none(string),
+          discv5Enabled = false,
+          discv5Config = defaultDiscoveryConfig,
+          discv5BootstrapNodes = newSeq[string](),
+          discv5BootstrapEnrs = newSeq[enr.Record](),
```

Comment:
Why two arguments here? Perhaps better to have only the `discv5BootstrapEnrs` as a `seq[enr.Record]` and leave it to the application to extract that from config.

Reply:
Addressed in [74c3622](https://github.com/waku-org/nwaku/pull/1512/commits/74c36223864222f0da14cee19f4a8f9b1f7f3654)